pkgs/development/python-modules/sjcl/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pycrypto
+}:
+
+buildPythonPackage rec {
+  pname = "sjcl";
+  version = "0.2.1";
+
+  format = "pyproject";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "3761ba01f9f049eaa2f67516054b7bfb2e27fe8bc7c9c630da7168fba58f0c57";
+  };
+
+  propagatedBuildInputs = [ pycrypto ];
+
+  doCheck = false;
+
```

```suggestion

  pythonImportsCheck = [
    "sjcl"
  ];
  
```

We also try to impor the modules that are documented, since it is another way to verify the package works.

---

added with [e119b23](https://github.com/NixOS/nixpkgs/pull/167951/commits/e119b230877703a025cd50aa9b317f686ab46dd6)